src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -243,6 +285,28 @@ public boolean intakeRightSeesNote() {
 		return intakeRightSensor.isPressed();
 	}
 
+	// debounce sensors
+	public boolean debouncedIntakeFrontSensor() {
+		// if (intakeFrontSensorDebouncer.calculate(intakeFrontSensor.isPressed())) {
+		// 	return true;
+		// }
+		return intakeFrontSensorDebouncer.calculate(intakeFrontSensor.isPressed());
+	}
+
+	public boolean debouncedIntakeLeftSensor() {
+		// if (intakeLeftSensorDebouncer.calculate(intakeLeftSensor.isPressed())) {
+		// 	return true;
+		// }
+		return intakeLeftSensorDebouncer.calculate(intakeFrontSensor.isPressed());
```

Comment:
For later (since we don't have the sensors on the robot right now): this should be checking `intakeLeftSensor.isPressed()` (same for the right sensor method below).